packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -198,10 +202,18 @@ class _ProductListPageState extends State<ProductListPage>
                       height: MediaQuery.of(context).size.height * .4,
                       package: AppHelper.APP_PACKAGE,
                     ),
-                    Text(
-                      appLocalizations.product_list_empty_title,
-                      style: themeData.textTheme.headlineLarge
-                          ?.apply(color: colorScheme.onBackground),
+                    const SizedBox(
+                      height: 20,
+                    ),
```

Comment:
* it should probably be a `Padding`
* you should use app sizes like `LARGE_SPACE`

Reply:
I will do the changes !!